Add process of expense reimbursement (#64)
hyphacoop/handbook: finance.md

After tax withholdings and other applicable deductions (e.g. CPP & EI employee contributions), the net amount is electronically transferred to the Member's bank account on pay day using **Wagepoint**.
Refer to the guides for [Payroll](/guides.md#payroll) to set up the payroll run for each pay period.

### Expense Reimbursement

> Member expenses eligible for reimbursement should be submitted to the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and receipts uploaded to [Employee Expenses](https://link.hypha.coop/receipts) in the same quarter they are incurred.

Reviewer: ```suggestion
Member expenses eligible for reimbursement should be submitted to the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and receipts uploaded to [Employee Expense Receipts](https://link.hypha.coop/receipts) in the same quarter they are incurred.
```
Name should better match link?

> Reimbursement amounts are paid out with wages on the next pay day, but does not appear on tax forms.

Reviewer: ```suggestion
Reimbursements are paid out with wages on the next pay day, but do not appear on tax forms.
```
